rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -98,6 +115,18 @@ def __init__(self, component_config: Optional[Dict[Text, Any]] = None) -> None:
             "default", self.module
         )
 
+    @classmethod
+    def validate_config(cls, config: Dict[Text, Any]) -> None:
+        """Validates that the component is configured properly."""
+        pass
```

Comment:
We could move the `_get_validated_model_url` here at some point (maybe not now for the sake of time), what do you think?

Reply:
Yep, I'll move that - that "getter" is just too strange anyway